src/pages/guideline_ios/1-01-foundation.md

```diff
@@ -4,17 +4,21 @@ title: Foundation
 date: 2019-01-11T00:15:32.894Z
 featuredImage: "/img/control-button.png"
 description: >-
-  SAP Fiori provides a consistent and holistic user experience for SAP software. By creating visually pleasing designs with a strong focus on ease of use, the experience is intuitive and simple, across all devices. With effortless interaction patterns, the SAP Fiori UX is designed for a powerful impact across your enterprise.   
+  SAP Fiori provides a consistent and holistic user experience for SAP software. By creating visually pleasing designs with a strong focus on ease of use, the experience is intuitive and simple, across all devices. With effortless interaction patterns, the SAP Fiori UX is designed for a powerful impact across your enterprise.
```

Comment:
This file should be used and `src/pages/guideline/ios/1-01-foundation.md` deleted